src/ReactDOM.re

```diff
@@ -477,6 +477,33 @@ external unmountComponentAtNode: Dom.element => unit =
 [@mel.module "react-dom"]
 external flushSync: (unit => unit) => unit = "flushSync";
 
+module Experimental = {
+  [@deriving jsProperties]
+  type preOptions = {
+    [@mel.as "as"] [@mel.optional]
+    _as: option([ | `script]),
+    [@mel.optional]
+    crossOrigin: option(string),
+    [@mel.optional]
+    integrity: option(string),
+    [@mel.optional]
+    nonce: option(string),
+  };
+
+  [@mel.module "react-dom"]
+  external preconnect: string => unit = "preconnect";
+  [@mel.module "react-dom"]
+  external prefetchDNS: string => unit = "prefetchDNS";
+  [@mel.module "react-dom"]
+  external preinit: string => unit = "preinit";
```

Comment:
in preinit the options are defined under https://react.dev/reference/react-dom/preinit#parameters, do you want to add them?

Reply:
Nice catch @davesnx , I missed `preinit` option.